db/migrate/20220125134340_create_bookings.rb

```diff
@@ -0,0 +1,8 @@
+class CreateBookings < ActiveRecord::Migration[6.1]
+  def change
+    create_table :bookings do |t|
+
```

Comment:
When we add a booking shouldn't we have user:references & lesson:references ?

Reply:
ok will fix this then re commit

Reply:
fixed!!